react/features/base/profile/middleware.js

```diff
@@ -2,7 +2,10 @@
 import { PROFILE_UPDATED } from './actionTypes';
 import MiddlewareRegistry from '../redux/MiddlewareRegistry';
```

Comment:
Please notice that we're importing from the redux feature in multiple lines!

````javascript
import MiddlewareRegistry from '../redux/MiddlewareRegistry';
import { toState } from '../redux';
````

That's why we group imports in the order third-party, other/outside feature, this/inside feature and separate the groups with empty lines. Let's please follow the coding style!

I'm taking care of this one.